packages/@aws-cdk/aws-sagemaker/lib/private/util.ts

```diff
@@ -11,3 +12,15 @@ export function hashcode(s: string): string {
   hash.update(s);
   return hash.digest('hex');
 }
+
+/**
+ * Whether two strings probably contain the same environment dimension (region or account).
+ *
+ * Used to compare either accounts or regions, and also returns true if both
+ * are unresolved (in which case both are expted to be "current region" or "current account").
+ * @param dim1 The first dimension to compare
+ * @param dim2 The second dimension to compare
+ */
+export function sameEnvDimension(dim1: string, dim2: string): boolean {
```

Comment:
Think we can just call this `sameEnv` or `inSameEnv`. Not in love with the word dimension since we don't use it elsewhere